src/main/java/org/junitpioneer/jupiter/resource/PathDeleter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.resource;
+
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+
+// visible for testing purposes
```

Comment:
I don't really understand this comment... The only way this class could be *less* visible if it were private, which is not even permitted? Comment should be removed, probably.

Reply:
Oh, great catch! I'd forgotten that top-level classes can never be `private`, so this comment is redundant. Fixed in https://github.com/junit-pioneer/junit-pioneer/pull/491/commits/5f1b12e8b25f39f94fb46e8c7bf2d1d11dcea8b4.